api/routes/api_sports.py

```diff
@@ -11,10 +12,15 @@ def handle_exception(e):
     response = {"error in service": str(e)}
     return get_error_response(response, 500)
 
+
 @sports_app.route('/all', methods=['GET'])
+@cache.cached(timeout=60*60)
+def get_all_sports_endpoint():
+    sports = get_all_sports(session)
+    return sports
+
+@sports_app.route('/league/<sport_type>', methods=['GET'])
+@cache.cached(timeout=60*60)
 def get_all_sports_endpoint():
-    try:
-        sports = get_all_sports()
-        return sports
-    except Exception as e:
-        print(e)
\ No newline at end of file
+    sports = 1#get_all_leagues_by_sport(session)
```

Comment:
should this be commented?

Reply:
so because the func doesn't work yet, it should be completed later in another task